implementingSwashbuckle.md

```diff
@@ -0,0 +1,133 @@
+# Documenting APIs with Swagger...sort of
```

Comment:
How about instead of "sort of" in the title, one of these?

"Documenting APIs with a generated Swagger.json file and fixing integration hiccups"
"A Pirate's Life for Me: Documenting APIs with Swagger and Swashbuckle"